apps/jobs/views.py

```diff
@@ -538,66 +540,93 @@ def get_submission_by_pk(request, submission_id):
 })
 @throttle_classes([UserRateThrottle,])
 @api_view(['PUT',])
-@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail, IsChallengeCreator,))
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail,))
 @authentication_classes((ExpiringTokenAuthentication,))
 def update_submission(request, challenge_pk):
     """
     API endpoint to update submission realted attributes
     """
```

Comment:
Can we please add a commented format of the data this API is going to receive, like what we have in submission worker. It will be helpful when we will look at this code in future.